spec/lib/radar_base_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe Radar::Base do
+  let(:radar) {
+    Radar::Base.new(RadarSetting.create())
+  }
+
+  # Test case for GH #591
+  it "should be do proper type casts" do
+    expect(radar.mark_missing(nil, [1,2,3])).to be_empty
```

Comment:
When I read this test case, I think to myself 'It should do proper type casts – how did @bascht test this?' Then I look at the code and I see 'Ok, he tests this by checking that the result is empty?!' and I'm confused.